app/views/layouts/.fuse_hidden00002b640000001b

```diff
@@ -0,0 +1,24 @@
+doctype html
```

Comment:
http://askubuntu.com/questions/493198/what-is-a-fuse-hidden-file-and-why-do-they-exist